File: assignment 17/p5.c
#include<stdio.h>
#include<stdlib.h>
int Frequency(int Arr[],int isize,int ino)
{
int icnt=0,counter=0;
for(icnt=0;icnt<isize;icnt++)
{
if((Arr[icnt]==ino))
{
counter++;
}
}
return counter;
}
int main()
{
int icnt=0,iret=0,ilength=0,ival=0;
int *ptr=NULL;
printf("enter no that frequency have to be calculated ");
scanf("%d",&ival);
printf("how many elements you want");
scanf("%d",&ilength);
ptr=(int*)malloc(sizeof(int)*ilength);
if(ptr==NULL)
{
printf("malloc fails ");
return -1;
}
printf("enter elements\n");
for(icnt=0;icnt<ilength;icnt++)
{
scanf("%d",&ptr[icnt]);
}
iret=Frequency(ptr,ilength,ival);
printf("frequency pf no is %d",iret);
free(ptr);
return 0;
}

File: assignment 32/P5.java
import java.util.*;
class Mangesh
{
	public void Display(int arr[],int ino)
	{
		int i=0,isize=0;
		isize=arr.length;
		for(i=0;i<isize;i++)
		{
			if(arr[i]%ino==0)
			{
				System.out.println(arr[i]);
			}
			
		}
		
	}
	
}
class P5
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		int ilen=0, icnt=0,ino=0;
		System.out.println("enter divisor");
		ino=sc.nextInt();
		System.out.println("how many elements you want");
		ilen=sc.nextInt();
		int brr[]=new int[ilen];
		System.out.println("enter no");
		for(icnt=0;icnt<ilen;icnt++)
		{
			brr[icnt]=sc.nextInt();
		}
		Mangesh mobj= new Mangesh();
	   System.out.println("");
		mobj.Display(brr,ino);
		
	}
}